<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>498</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>498</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>亚马逊棋</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>120</width>
    <height>120</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="newGameFirst">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>215</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>新游戏(先手)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadGame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>430</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>载入游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitGame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>430</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="QLabel" name="textTitle">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>70</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Adobe Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>亚  马  逊  棋</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="newGameSecond">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>190</y>
      <width>215</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>新游戏(后手)</string>
    </property>
   </widget>
   <widget class="QLabel" name="stepShowSelf">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>550</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stepShowCom">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>550</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Made By FlierKing</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
